### DISCUSSION TITLE
Announcement URLFrontier 2

### DISCUSSION THREAD

**Discussion by jnioche** [crawler-commons/url-frontier#43] [General]
I am pleased to announce that **URLFrontier 2** is now officially a new project funded by NLNet as part of the NGI Zero Discovery fund.
You can find a [roadmap on the wiki](https://github.com/crawler-commons/url-frontier/wiki/Roadmap-URLFrontier-2) summarising the four milestones of the project.

I will keep you updated on the progress of the project. As usual, everyone is welcome to get involved one way or another.

**Comment by lewismc** [comment]
Exciting stuff Julien. Congratulations. This project direction is very
promising.

On Thu, Jan 20, 2022 at 02:55 Julien Nioche ***@***.***>
wrote:

> I am pleased to announce that *URLFrontier 2.0* is now officially a new
> project funded by NLNet as part of the NGI Zero Discovery fund.
> You can find a roadmap on the wiki
> <https://github.com/crawler-commons/url-frontier/wiki/Roadmap-URLFrontier-2.0>
> summarising the four milestones of the project.
>
> I will keep you updated on the progress of the project. As usual, everyone
> is welcome to get involved one way or another.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/crawler-commons/url-frontier/discussions/43>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAI4TF7XNQ7S74XNX3BXAY3UW7SZXANCNFSM5MMNCJWQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 

*Lewis*
Dr. Lewis J. McGibbney Ph.D, B.Sc
*Skype*: lewis.john.mcgibbney
